docs/source/reference/settings_example.py

```diff
@@ -89,6 +89,8 @@
 
 ROOT_URLCONF = 'myportal.urls'
 
+# Enable newest DGPF template styles
+BASE_TEMPLATES = 'globus-portal-framework/v3/'
```

Comment:
We probably don't want to include this, and allow the default template version drive the version. The plan was for DGPF v0.5.x to switch from v2 to v3 templates, and document how portals should upgrade. `BASE_TEMPLATES` would only need to be specified if a portal wanted to upgrade to the newer version and use older templates (Or custom templates, if they created their own).